client_app/src/components/volunteer/VolunteerDashboard.js

```diff
@@ -23,20 +30,21 @@ function VolunteerDashboard() {
         <h3>Impact Created</h3>
         <div className="card">
           <h4>Total hours served</h4>
-          <p>106</p>
+          <p>{impactData.totalHours}</p>
         </div>
         <div className="card">
           <h4>Lives Touched</h4>
-          <p>17</p>
+          <p>Too many to count</p>
         </div>
         <div className="card">
           <h4>Shelters served</h4>
-          <p>4</p>
+          <p>{impactData.sheltersServed}</p>
         </div>
       </div>
       <div className="cta-button">
         <button onClick={signupClick}>Sign up for shifts</button>
       </div>
+      <PastShifts onImpactDataUpdate={(data) => setImpactData(data)} />
```

Comment:
This change places "Past Shifts" into the dashboard. I don't think we want to do that.
<img width="1533" alt="Screenshot 2025-01-30 at 3 20 42 PM" src="https://github.com/user-attachments/assets/e901dfe6-7e10-4d39-9588-fbfdb754c91b" />